code/modules/plumbing/plumbers/reaction_chamber.dm

```diff
@@ -86,18 +87,21 @@
 	if(.)
 		return
 	switch(action)
+		if("add")
+			var/selected_reagent = tgui_input_list(usr, "Select Reagent", "Reagent", GLOB.chemical_name_list)
```

Comment:
Do most tgui_input_list do "Select Reagent" instead of "Select reagent"?

Reply:
I'll change it to Select reagent